## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 447538 --extra-nixpkgs-config '{ nvidia.acceptLicense = true; }' --checkout commit --skip-package-regex .+_libre.+`
Commit: `3697b7c937ea4b2b9ef02ab7ffd3a1ef8c3196c4`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 24 packages marked as broken and skipped:</summary>
  <ul>
    <li>haskellPackages.Obsidian</li>
    <li>haskellPackages.Obsidian.doc</li>
    <li>haskellPackages.accelerate-bignum</li>
    <li>haskellPackages.accelerate-bignum.doc</li>
    <li>haskellPackages.accelerate-blas</li>
    <li>haskellPackages.accelerate-blas.doc</li>
    <li>haskellPackages.accelerate-cublas</li>
    <li>haskellPackages.accelerate-cublas.doc</li>
    <li>haskellPackages.accelerate-cuda</li>
    <li>haskellPackages.accelerate-cuda.data</li>
    <li>haskellPackages.accelerate-cuda.doc</li>
    <li>haskellPackages.accelerate-cufft</li>
    <li>haskellPackages.accelerate-cufft.doc</li>
    <li>haskellPackages.accelerate-examples</li>
    <li>haskellPackages.accelerate-examples.doc</li>
    <li>haskellPackages.accelerate-fft</li>
    <li>haskellPackages.accelerate-fft.doc</li>
    <li>haskellPackages.accelerate-fourier-benchmark</li>
    <li>haskellPackages.accelerate-fourier-benchmark.doc</li>
    <li>haskellPackages.accelerate-kullback-liebler</li>
    <li>haskellPackages.accelerate-llvm-ptx</li>
    <li>haskellPackages.accelerate-llvm-ptx.doc</li>
    <li>haskellPackages.wigner-ville-accelerate</li>
    <li>haskellPackages.wigner-ville-accelerate.doc</li>
  </ul>
</details>
<details>
  <summary>:x: 28 packages failed to build:</summary>
  <ul>
    <li>haskellPackages.cublas</li>
    <li>haskellPackages.cublas.doc</li>
    <li>haskellPackages.cuda</li>
    <li>haskellPackages.cuda.doc</li>
    <li>haskellPackages.cufft</li>
    <li>haskellPackages.cufft.doc</li>
    <li>haskellPackages.cusolver</li>
    <li>haskellPackages.cusolver.doc</li>
    <li>haskellPackages.cusparse</li>
    <li>haskellPackages.cusparse.doc</li>
    <li>haskellPackages.libnvvm</li>
    <li>haskellPackages.libnvvm.doc</li>
    <li>haskellPackages.nvvm</li>
    <li>haskellPackages.nvvm.doc</li>
    <li>linuxKernel.packages.linux_5_10.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_5_15.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_5_4.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_6_1.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_6_12.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_hardened.nvidia_x11_beta_open (linuxKernel.packages.linux_6_12_hardened.nvidia_x11_beta_open)</li>
    <li>linuxKernel.packages.linux_6_16.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_6_6.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_lqx.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_xanmod.nvidia_x11_beta_open</li>
    <li>linuxKernel.packages.linux_xanmod_latest.nvidia_x11_beta_open (linuxKernel.packages.linux_xanmod_stable.nvidia_x11_beta_open)</li>
    <li>linuxKernel.packages.linux_zen.nvidia_x11_beta_open</li>
    <li>xpraWithNvenc</li>
    <li>xpraWithNvenc.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 68 packages built:</summary>
  <ul>
    <li>bottles</li>
    <li>bottles-unwrapped</li>
    <li>bumblebee</li>
    <li>gl-gsync-demo</li>
    <li>gwe</li>
    <li>libvibrant</li>
    <li>linuxKernel.packages.linux_5_10.nvidia_x11</li>
    <li>linuxKernel.packages.linux_5_10.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_5_10.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_5_10.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_5_15.nvidia_x11</li>
    <li>linuxKernel.packages.linux_5_15.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_5_15.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_5_15.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_5_4.nvidia_x11</li>
    <li>linuxKernel.packages.linux_5_4.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_5_4.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_5_4.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_6_1.nvidia_x11</li>
    <li>linuxKernel.packages.linux_6_1.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_6_1.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_6_1.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_6_12.nvidia_x11</li>
    <li>linuxKernel.packages.linux_6_12.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_6_12.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_6_12.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_hardened.nvidia_x11 (linuxKernel.packages.linux_6_12_hardened.nvidia_x11)</li>
    <li>linuxKernel.packages.linux_hardened.nvidia_x11.bin (linuxKernel.packages.linux_6_12_hardened.nvidia_x11.bin)</li>
    <li>linuxKernel.packages.linux_hardened.nvidia_x11.firmware (linuxKernel.packages.linux_6_12_hardened.nvidia_x11.firmware)</li>
    <li>linuxKernel.packages.linux_hardened.nvidia_x11.lib32 (linuxKernel.packages.linux_6_12_hardened.nvidia_x11.lib32)</li>
    <li>linuxKernel.packages.linux_6_16.nvidia_x11</li>
    <li>linuxKernel.packages.linux_6_16.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_6_16.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_6_16.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_6_6.nvidia_x11</li>
    <li>linuxKernel.packages.linux_6_6.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_6_6.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_6_6.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_lqx.nvidia_x11</li>
    <li>linuxKernel.packages.linux_lqx.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_lqx.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_lqx.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_xanmod.nvidia_x11</li>
    <li>linuxKernel.packages.linux_xanmod.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_xanmod.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_xanmod.nvidia_x11.lib32</li>
    <li>linuxKernel.packages.linux_xanmod_latest.nvidia_x11 (linuxKernel.packages.linux_xanmod_stable.nvidia_x11)</li>
    <li>linuxKernel.packages.linux_xanmod_latest.nvidia_x11.bin (linuxKernel.packages.linux_xanmod_stable.nvidia_x11.bin)</li>
    <li>linuxKernel.packages.linux_xanmod_latest.nvidia_x11.firmware (linuxKernel.packages.linux_xanmod_stable.nvidia_x11.firmware)</li>
    <li>linuxKernel.packages.linux_xanmod_latest.nvidia_x11.lib32 (linuxKernel.packages.linux_xanmod_stable.nvidia_x11.lib32)</li>
    <li>linuxKernel.packages.linux_zen.nvidia_x11</li>
    <li>linuxKernel.packages.linux_zen.nvidia_x11.bin</li>
    <li>linuxKernel.packages.linux_zen.nvidia_x11.firmware</li>
    <li>linuxKernel.packages.linux_zen.nvidia_x11.lib32</li>
    <li>mangohud</li>
    <li>mangohud.doc</li>
    <li>mangohud.man</li>
    <li>mangojuice</li>
    <li>monitor</li>
    <li>nero-umu</li>
    <li>nixpkgs-manual</li>
    <li>nvfancontrol</li>
    <li>primus</li>
    <li>tuxclocker</li>
    <li>tuxclocker-nvidia-plugin</li>
    <li>tuxclocker-plugins-with-unfree</li>
    <li>vibrantlinux</li>
    <li>xfce.xfce4-sensors-plugin</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>haskellPackages.cuda</summary>
<pre>/nix/store/8av8pfs7bnyc6hqj764ns4z1fnr9bva1-gnutar-1.35/bin/tar
No uhc found
checking for environment variable CUDA_PATH
Path accepted: /nix/store/jkva371d2pm90mp05xs0dassah1qr2r3-cuda-merged-12.8
Found CUDA toolkit at:
/nix/store/jkva371d2pm90mp05xs0dassah1qr2r3-cuda-merged-12.8 (set CUDA_PATH to
override this)
Storing parameters to cuda.buildinfo.generated
Using build information from 'cuda.buildinfo.generated'.
Provide a 'cuda.buildinfo' file to override this behaviour.
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Using build information from 'cuda.buildinfo.generated'.
Provide a 'cuda.buildinfo' file to override this behaviour.
Preprocessing library for cuda-0.11.0.1..
c2hs: Errors during expansion of binding hooks:

src/Foreign/CUDA/Driver/Graph/Capture.chs:156: (column 15) [ERROR]  >>> Unknown identifier!
  Cannot find a definition for `cuStreamGetCaptureInfo' in the header file.
</pre>
</details>
<details>
<summary>xpraWithNvenc</summary>
<pre>
/nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/bits/stdio2.h(305): error: linkage specification is incompatible with previous "fgets" (declared at line 654 of /nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/stdio.h)
  fgets (char * __restrict const __attribute__ ((__pass_object_size__ (2 > 1))) __s, int __n,
  ^

/nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/bits/stdio2.h(322): error: linkage specification is incompatible with previous "fread" (declared at line 728 of /nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/stdio.h)
  fread (void * __restrict const __attribute__ ((__pass_object_size__ (2 > 1))) __ptr,
  ^

/nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/bits/stdio2.h(342): error: linkage specification is incompatible with previous "fgets_unlocked" (declared at line 677 of /nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/stdio.h)
  fgets_unlocked (char * __restrict const __attribute__ ((__pass_object_size__ (2 > 1))) __s,
  ^

/nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/bits/stdio2.h(362): error: linkage specification is incompatible with previous "fread_unlocked" (declared at line 756 of /nix/store/gf3wh0x0rzb1dkx0wx1jvmipydwfzzd5-glibc-2.40-66-dev/include/stdio.h)
  fread_unlocked (void * __restrict const __attribute__ ((__pass_object_size__ (0))) __ptr,
  ^

21 errors detected in the compilation of "fs/share/xpra/cuda/RGBX_to_RGB.cu".

failed to rebuild the cuda kernels ['XRGB_to_NV12', 'XRGB_to_YUV444', 'BGRX_to_NV12', 'BGRX_to_YUV444', 'BGRX_to_RGB', 'RGBX_to_RGB', 'RGBA_to_RGBAP', 'BGRA_to_RGBAP']</pre>
</details>
</details>

--- a/pkgs/os-specific/linux/nvidia-x11/default.nix
+++ b/pkgs/os-specific/linux/nvidia-x11/default.nix
@@ -72,12 +72,12 @@ rec {
   stable = if stdenv.hostPlatform.system == "i686-linux" then legacy_390 else production;
 
   production = generic {
-    version = "580.82.09";
-    sha256_64bit = "sha256-Puz4MtouFeDgmsNMKdLHoDgDGC+QRXh6NVysvltWlbc=";
-    sha256_aarch64 = "sha256-6tHiAci9iDTKqKrDIjObeFdtrlEwjxOHJpHfX4GMEGQ=";
-    openSha256 = "sha256-YB+mQD+oEDIIDa+e8KX1/qOlQvZMNKFrI5z3CoVKUjs=";
-    settingsSha256 = "sha256-um53cr2Xo90VhZM1bM2CH4q9b/1W2YOqUcvXPV6uw2s=";
-    persistencedSha256 = "sha256-lbYSa97aZ+k0CISoSxOMLyyMX//Zg2Raym6BC4COipU=";
+    version = "580.95.05";
+    sha256_64bit = "sha256-hJ7w746EK5gGss3p8RwTA9VPGpp2lGfk5dlhsv4Rgqc=";
+    sha256_aarch64 = "sha256-zLRCbpiik2fGDa+d80wqV3ZV1U1b4lRjzNQJsLLlICk=";
+    openSha256 = "sha256-RFwDGQOi9jVngVONCOB5m/IYKZIeGEle7h0+0yGnBEI=";
+    settingsSha256 = "sha256-F2wmUEaRrpR1Vz0TQSwVK4Fv13f3J9NJLtBe4UP2f14=";
+    persistencedSha256 = "sha256-QCwxXQfG/Pa7jSTBB0xD3lsIofcerAWWAHKvWjWGQtg=";
   };
 
   latest = selectHighestVersion production (generic {

